### DISCUSSION TITLE
Execute all templates available.

### DISCUSSION THREAD

**Discussion by righettod** [projectdiscovery/nuclei-templates#9577] [Q&A] [answered]
Hi,

Does the command `nuclei -u test.com` run all available templates or do I need to add a specific instruction?

Please feel free to redirect me to the documentation if I missed anything while reading it.

Many thanks in advance for your help 😃

**Comment by rxerium** [comment, marked as answer]
Hey @righettod no not all, there are some exceptions listed in the [Nuclei Ignore](https://github.com/projectdiscovery/nuclei-templates/blob/main/.nuclei-ignore) file, removing them from this file would include them in the default scan :)

**Reply by righettod** [reply]
Thanks a lot @rxerium for the clarification 👍
